sdk/spring/migration-guide.md

```diff
@@ -147,6 +147,136 @@ For a full list of common configurations, check this list **[placeholder]**.
 
 ### Each SDK configurations
 
+## Migration steps of each SDK   
```

Comment:
This whole section is for configuration changes, there's one section about artifact changes above.

Reply:
I think it shoud not under  `## Configuration properties` section, 
I don't know where to put it, so I create a new section `## Migration steps of each SDK`.